src/main/java/com/urbanairship/connect/client/model/StreamQueryDescriptor.java

```diff
@@ -24,6 +24,7 @@
     private final Creds creds;
     private final Set<Filter> filters;
     private final Optional<Subset> subset;
+    private final Optional<Boolean> offsetUpdatesEnabled;
```

Comment:
is Optional.empty intended to mean something different from false here? Does something rely on whether or not the client set it?

Reply:
I originally used a bare boolean, but decided to use an Optional since the field is in fact optional on the request.

Reply:
doesn't Optional.empty == false for all intents and purposes?

Reply:
It means use the server default, that that's `false` right now strikes me as a separate concern.